src/api/app/views/person/token/index.xml.builder

```diff
@@ -1,6 +1,6 @@
 xml.directory(count: @list.length) do |dir|
   @list.each do |token|
-    p = { id: token.id, string: token.string, kind: token.token_name, name: token.name, triggered_at: token.triggered_at }
+    p = { id: token.id, string: token.string, kind: token, name: token.name, triggered_at: token.triggered_at }
```

Comment:
TBH, I don't like that we take a field and transform it to `to_s` if it this field isn't really either the name or the token type.

Reply:
I don't like using this also. `Token.token_name` is wrong, but using `token.to_s` is wrong also. I would go by naming to `token_name` as `token_kind` to make it explicit what are we talking about.

Reply:
Note that before we had `Token.token_name` AND `Token#token_name` they were consistent.  The user defined name was introduced later unfortunately as "name". So now we have `Token#name`, and `Token.name` meaning totally different things..

We have already `Token#type`, `Token.token_type`, `Token.name`, `Token#to_s`.. i think `Token#kind` is one more variation for confusion... maybe `Token#operation`? `Token#service` could work too, but since we have already `Token::Service´ that would be confusing too.. 

I personally don't have anything against `Token#token_name` though :)

Reply:
What we call `token_name` is, in fact, the token type but in a humanized way. What about `humanized_token_type`?

Reply:
Hm, I think we don't want to express the token type but the service being triggered. `service`, `trigger_service`  or `service_type` would be IMO better, but then we have already a `Token::Service` which somehow could lead to confusion too